Automattic/wp-calypso
# Nav Redesign: Remove notifications pointer

## Proposed Changes

Removes the notifications pointer added in https://github.com/Automattic/wp-calypso/pull/90495 since it was designed to show up only before May 23.

## Why are these changes being made?

To clean up obsolete code.

## Testing Instructions

This is actually untestable since the notifications pointer is no longer visible, so just use the Calypso live below, go to `/sites` and make sure there are no regressions.

## Pre-merge Checklist
